packages/indy-vdr/src/pool/IndyVdrPoolService.ts

```diff
@@ -188,7 +188,7 @@ export class IndyVdrPoolService {
       this.logger.trace(`Retrieved did '${did}' from ledger '${pool.indyNamespace}'`, result)
 
       return {
-        did: result,
+        did: { nymResponse: { did: result.dest, verkey: result.verkey }, indyNamespace: pool.indyNamespace },
```

Comment:
Why is this change needed? We can already extract the namespace from the pool (which is so returned). Should we make `nymResponse` a top-level property?

Reply:
This was added just to match `CachedDidResponse` interface that is part of the `PublicDidRequest` supposed to be returned by `getDidFromPool()` (otherwise, it wouldn't properly process the GET_NYM response). For the moment it seems to be only used internally, so probably it would be good to refactor it a bit to match the usage within the wonderful indy-vdr world.